kratos/sources/kratos_parameters.cpp

```diff
@@ -483,6 +483,19 @@ bool Parameters::RemoveValue(const std::string& rEntry)
 /***********************************************************************************/
 /***********************************************************************************/
 
+bool Parameters::RemoveValues(const std::vector<std::string>& rEntries)
+{
+    bool sucess;
+    for (auto& r_entry : rEntries) {
+        sucess = this->RemoveValue(r_entry);
+        if (!sucess) return false;
+    }
+    return true;
+}
```

Comment:
This removal is order dependent from the `rEntries`. If a mid item is not removed successfully, it will exit having `Parameters` removed half of the entries and we let the user of this function to decide on to throw an error or not.
I would do it in either of the following ways.
1. First check whether all the `rEntries` present in `Parameters`, if so then removing all.
2. do not return if the item was not successfully removed, return `success` at the end. Example: `bool sucess = true; success = success && this->RemoveValue(r_entry); return success` in respective lines.

Reply:
Okay

Reply:
Done